include/torch-mlir/Conversion/Utils/Utils.h

```diff
@@ -79,6 +79,13 @@ SmallVector<Value> getTypeConvertedValues(OpBuilder &b, Location loc,
                                           TypeConverter *converter,
                                           SmallVectorImpl<Value> &vs);
 
+static constexpr int64_t kDynamicSize = -1;
```

Comment:
Use `kUnknownSize` instead of hardcoding it as `-1`.

Reply:
`static constexpr int64_t kDynamicSize = kUnknownSize;`

Reply:
Done